Project/app/build.gradle

```
@@ -26,6 +26,17 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
         }
     }
+    compileOptions {
+        sourceCompatibility = 1.8
+        targetCompatibility = 1.8
+    }
+
+    kotlinOptions {
+        jvmTarget = "1.8"
+    }
+    testOptions {
+        unitTests.returnDefaultValues = true
```

In general it's not recommended to have `returnDefaultValues` set to true. It might hide errors in the tests or make it more difficult to track problems.

```suggestion
        unitTests.returnDefaultValues = false
```